### DISCUSSION TITLE
第6节课作业(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#444] [作业]
第6节课作业(19班)

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135724355

**Comment by Sixdes** [comment]
https://github.com/Sixdes/puyu-course/tree/main/homework0122

**Comment by jts250** [comment]
https://github.com/jts250/homework_llm/blob/main/homework6.pdf

**Comment by romanwin28** [comment]
大模型评测：
[OpenCompass 大模型评测.pdf](https://github.com/InternLM/tutorial/files/14035201/OpenCompass.pdf)

**Comment by Jiangzzzzl** [comment]
<img width="878" alt="2a411bf146e8cc288c2e18d1c665352" src="https://github.com/InternLM/tutorial/assets/150671792/c4796151-32aa-4ba0-97f6-f6d9c4ee56f6">
<img width="878" alt="aefcf48b34747587f262e92568c7c0d" src="https://github.com/InternLM/tutorial/assets/150671792/de4b9d69-5a52-4671-bb34-7828d35d7d84">
<img width="1161" alt="1706110225705" src="https://github.com/InternLM/tutorial/assets/150671792/115ff5a8-ce00-478b-abc9-41f04f815f8b">

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/06/asgmt06.md

**Comment by mkdirmushroom** [comment]
[第6节课作业](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial5_assignment.md)

**Comment by Hamasuta666** [comment]
第6节课笔记（作业在文末）：https://zhuanlan.zhihu.com/p/681111451
